### PR TITLE
Feature/ebooks

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

File: code/SiteBuilder.ts [hunk 1]
```diff
@@ -50,25 +50,25 @@ const buildImportsFile = (filepath, varMap) => {
   Disk.write(
     filepath,
     `importOnly\n` +
-      Object.keys(varMap)
-        .map(key => {
-          let value = varMap[key]
+    Object.keys(varMap)
```
Comment (breck7) [inline code/SiteBuilder.ts:53]: this file should not be changed or included in this PR

File: site/lists/lists.scroll [hunk 2]
```diff
@@ -16,6 +16,7 @@ pipeTable
  File Extensions|./extensions.html
  Programming Language Creators|./creators.html
  Job Sites|./jobs.html
+ Ebooks|./resources.html
```
Comment (breck7) [inline site/lists/lists.scroll:19]: change to "Free ebooks about programming" and change permalink to `ebooks.html`

File: site/lists/resources.scroll [hunk 3]
```diff
@@ -0,0 +1,12 @@
+title Programming languages ebooks
```
Comment (breck7) [inline site/lists/resources.scroll:1]: title Free eBooks about programming

File: site/lists/resources.scroll [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import settings.scroll
+columnWidth 100
+
+* List of programming languages ebooks
```
Comment (breck7) [inline site/lists/resources.scroll:5]: you can delete this line

File: site/lists/resources.scroll [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+groups index
+endSnippet
+
+import resourcesData.scroll
```
Comment (breck7) [inline site/lists/resources.scroll:10]: great idea to separate the data from the page. i would just rename the file to `ebooksData.scroll`

File: site/lists/resourcesData.scroll [hunk 6]
```diff
@@ -0,0 +1,7 @@
+importOnly
+commaTable
+ series,seriesLink,publisher,year
+ Introduction to the Theory of Programming Languages,https://bertrandmeyer.com/2022/09/28/introduction-theory-programming-languages-full-book-now-freely-available/, Bertrand Meyer,2022
```
Comment (breck7) [inline site/lists/resourcesData.scroll:4]: you have an extra space: "/, B"

File: site/lists/resourcesData.scroll [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+ Github EbookFoundation,https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-subjects.md#computer-science, EbookFoundation
```
Comment (breck7) [inline site/lists/resourcesData.scroll:5]: Only link *directly* to pdfs or single book entries. No links to lists of ebooks. Just links directly to the individual books

File: site/lists/resourcesData.scroll [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+ w3schools,https://www.w3schools.com/, w3schools
+ guru99,https://www.guru99.com/, guru99
```
Comment (breck7) [inline site/lists/resourcesData.scroll:7]: remove

File: site/lists/resourcesData.scroll [hunk 9]
```diff
@@ -0,0 +1,7 @@
+importOnly
+commaTable
+ series,seriesLink,publisher,year
+ Introduction to the Theory of Programming Languages,https://bertrandmeyer.com/2022/09/28/introduction-theory-programming-languages-full-book-now-freely-available/, Bertrand Meyer,2022
+ Github EbookFoundation,https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-subjects.md#computer-science, EbookFoundation
+ w3schools,https://www.w3schools.com/, w3schools
```
Comment (breck7) [inline site/lists/resourcesData.scroll:6]: remove

**Review by breck7 — CHANGES_REQUESTED** [review]
getting closer!

**Comment by celtic-coder** [comment]
Good work, Kimera (@rkimera94)!

**Pull request merged**